src/Domain/DomainHelpers.cpp

```diff
@@ -310,7 +310,7 @@ std::vector<std::array<size_t, two_to_the(VolumeDim)>> corners_from_two_maps(
         gsl::at(corners_for_block2, vci_map2.local_corner_number()) =
             vci_map1.local_corner_number();
         break;
-      } else {
+      } else {  // NOLINT(llvm-else-after-return)
```

Comment:
llvm has a fair point here, don't you think? I understand if you don't want to refactor this code though. Instead of the NOLINT we can just merge and ignore the clang-tidy complaint.

Reply:
I don't understand what LLVM is trying to tell me. As far as I can tell it wants me to remove the `break;` but that actually changes the control flow.

Reply:
I think it wants you to delete the `else` scope, moving the code one indentation level down. The `else` doesn't really do anything here because the `break` in the `if` means the code below doesn't run anyway when the `if` is triggered.

Reply:
Ah, I'll try that and see if it works.

Reply:
Looks like a rename of readability-else-after-return, which is in our exclusion list.